website/content/docs/release-notes/nomad/upcoming.mdx

```diff
@@ -28,6 +28,10 @@ unless there's a required change due to a security vulnerability.
 * Nomad will remove support for HCL1 job specifications and the `-hcl1` flag on
   the `nomad job run` and other commands. Refer to [GH-20195][] for more details.
 * Nomad will remove the [`tls_prefer_server_cipher_suites`][] agent configuration.
+* Nomad will remove support for Nomad client agents older than 1.6.0. Older
+  nodes will fail heartbeats. Nomad servers will mark the workloads on those
+  nodes as lost and reschedule them normally according to the job's `reschedule`
```

Comment:
Do we want to add a link here to the reschedule block?

Reply:
Good call. Done.